tests/test_io.py

```diff
@@ -1931,51 +1932,77 @@ def _test_result_exception(self, result_type, result_file_name="", fmi_type="me"
             with nose.tools.assert_raises(ResultSizeError):
                 res = model.simulate(options=opts)
 
-        def _test_result_size_verification(self, result_type, result_file_name=""):
+        def _test_result_size_verification(self, result_type, result_file_name="", dynamic_diagnostics=False):
```

Comment:
Can we maybe add a docstring what is being tested here?

Reply:
Added.